k9mail/src/main/java/com/fsck/k9/controller/MessagingController.java

```diff
@@ -544,24 +540,12 @@ public void messageFinished(LocalMessage message, int number, int ofTotal) {
                 }
             };
 
-            // alert everyone the search has started
-            if (listener != null) {
-                listener.listLocalMessagesStarted(account, null);
-            }
-
             // build and do the query in the localstore
             try {
                 LocalStore localStore = account.getLocalStore();
                 localStore.searchForMessages(retrievalListener, search);
             } catch (Exception e) {
-                if (listener != null) {
-                    listener.listLocalMessagesFailed(account, null, e.getMessage());
```

Comment:
Hmm. I think we'll probably want to re-add this sort of thing. But maybe under a separate ErrorListener. For now it's fine.